book/Zapp-Pipes/3.-Native-bridge-API.md

```diff
@@ -11,10 +11,14 @@ The `nativeBridge` object provides the following methods :
   * `accountId`: Zapp account Id
   * `broadcasterId`: Applicaster broadcaster Id
   * `bundleIdentifier`: bundle identifier of the app
-  * `platform`: android or ios
+  * `platform`: *android* or *ios*
   * `apiSecretKey`: api private key
   * `languageLocale`: language locale configured in the device
   * `countryLocale`: country locale configured in the device
+  * `advertisingIdentifier`: this returns the *idfa* for iOS and *aaid* for Android
```

Comment:
I suggest adding a flag called isAdvertisingIdEnabled.

Reply:
If advertisement is not enabled, the advertisingIdentifier will return no value. So it is already implicit.

Reply:
Cool